Editor/Resources/EditorWindow/Pages/AwsUserProfilesPage.uxml

```diff
@@ -0,0 +1,136 @@
+        <!--            <ui:Button text="X" name="CloseInfoBox"/>-->
+        <!--        </ui:VisualElement>-->
+        <!--        <ui:VisualElement name="ErrorInfoBox" class="InfoBox ErrorBox Tab2Error">-->
+        <!--            <ui:VisualElement/>-->
+        <!--            <ui:Label text="S3 Buckets reached maximum limit 100, manage your list of buckets" name="Label"/>-->
+        <!--            <ui:Button text="View S3 Console \uf08e"/>-->
+        <!--            <ui:Button text="X" name="CloseInfoBox"/>-->
+        <!--        </ui:VisualElement>-->
+        <ui:VisualElement name="Cards" class="separator separator--large separator--horizontal">
+            <ui:VisualElement name="NoAccountCard" class="card card--large card--centered separator">
+                <ui:Image name="UserProfilePageAccountCardNoAccountIcon" class="card__icon card__icon--large"/>
+                <ui:Label name="UserProfilePageAccountCardNoAccountTitle" class="card__title" text="I don’t have an AWS account"/>
+                <ui:Label name="UserProfilePageAccountCardNoAccountDescription" text="[text about needing an AWS account and it can take a maximum of two sentences]"/>
+                <ui:VisualElement class="external-link">
+                    <ui:Label name="UserProfilePageAccountCardNoAccountLink" text="Learn More"/>
+                    <ui:Image name="ExternalLinkIcon" class="icon--small"/>
+                </ui:VisualElement>
+                <ui:Button name="UserProfilePageAccountCardNoAccountButton" class="button">
+                    <ui:Label name="UserProfilePageAccountCardNoAccountButtonLabel" text="Create an AWS Account"/>
+                    <ui:Image name="ExternalLinkIcon" class="icon--small"/>
+                </ui:Button>
+            </ui:VisualElement>
+            <ui:VisualElement name="HaveAccountCard" class="card card--large card--centered separator">
+                <ui:Image name="UserProfilePageAccountCardNewAccountIcon" class="card__icon card__icon--large"/>
+                <ui:Label name="UserProfilePageAccountCardNewAccountTitle" class="card__title" text="I have an AWS account"/>
+                <ui:Label name="UserProfilePageAccountCardNewAccountDescription" text="[text about having an AWS account and it can take a maximum of two sentences]"/>
+                <ui:Button name="UserProfilePageAccountCardHasAccountButton" class="button button--primary" text="Add New Profile"/>
+            </ui:VisualElement>
+        </ui:VisualElement>
+        <ui:VisualElement name="AddNewProfile" class="separator separator--large">
+            <ui:Label name="UserProfilePageAccountNewProfileTitle" class="page__subtitle" text="AWS Account Details"/>
+            <ui:VisualElement>
+                <ui:VisualElement name="AWSAccountProfile" class="form-row">
+                    <ui:Label name="UserProfilePageAccountNewProfileName" class="form-row__label" text="AWS Account Profile"/>
```

Comment:
While refactoring this file could you please make element name consistent with what we've been following? 
Some recommendations:

1. Use consistent prefixes. The next line just reads `AWSAccountProfile` but should perhaps have been `UserProfilePageAccountNewProfileInput` based on what we've been following.
2. Should this element have a name `UserProfilePageAccountNewProfileLabel` instead? 

We should ensure all elements follow similar patterns. Please review the entire file to make sure this is followed.

Reply:
Renamed names to be consistant on page

Reply:
Hey Kyle, thanks for making these updates. We discussed this page would be split up into smaller components? Is there any update on the status of that task?

Reply:
I have pulled the creation form elements into their own uxml